EnglishFilter/sections/whitelist_stealth.txt

```diff
@@ -32,6 +32,8 @@
 !#########################################
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/78197
+@@||cloudfront.net^$domain=postype.com,stealth

```

Comment:
Please specify in comment which of stealth mode options causes the issue. 
(like in comment below)

Reply:
btw it seems we can narrow this rule by $image modifier

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/78201/commits/66e15d9d5bea9044460be7c9fb17ea1f06199317